src/Header.svelte

```diff
@@ -27,14 +27,9 @@
   export let config: Config;
 
   let query: string;
-  let results: ResolvedSearch;
+  let results: ProjectsAndProfiles | undefined = undefined;
```

Comment:
I think this could also be done with using `null` instead of `undefined` as used in other components in the codebase

```suggestion
  let results: ProjectsAndProfiles | null = null;
```

Reply:
2697b8a7cd213c66eda38f2485c59683b6571cb9